common/graphical_culture/fe_ship_colors_note.log

```diff
@@ -0,0 +1,5 @@
+# FALLEN EMPIRE TYPES BY GRAPHICAL CULTURE #
```

Comment:
This was a reference file, not something that will actually be seen in the game, by its code or otherwise. I kept forgetting what FE ship set matched what number, so I added this in order to provide consistency for FE nations. It was very valuable to me.